packages/asset-service/src/service/AssetService.ts

```diff
@@ -1,9 +1,64 @@
+import { caip19 } from '@shapeshiftoss/caip'
 import { Asset, AssetDataSource, BaseAsset, ChainTypes, NetworkTypes } from '@shapeshiftoss/types'
 import axios from 'axios'
 
+/* eslint-disable */
+const Identicon = require('identicon.js')
+const { fonts, renderPixels } = require('js-pixel-fonts')
+/* eslint-enable */
+
 import assetsDescriptions from './descriptions.json'
 import localAssetData from './generatedAssetData.json'
 
+const getRenderedIdenticonB64 = (symbol: string, tokenId: string): string => {
+  // `imageSize` defines the size (px) of generated identicon. It can be modified according to needs.
+  const imageSize = 128
```

Comment:
What do you guys think about making this configurable? Might be too big in some cases

Reply:
The latest commit makes the size configurable